client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -124,6 +134,18 @@ class CustomContentTypes extends Component {
 		);
 	}
 
+	renderBlogPostSettings() {
+		const { translate } = this.props;
+		const fieldLabel = translate( 'Blog posts' );
```

Comment:
Nice tip @a8ci18n, but we're going with the lowercase one intentionally - it was a design decision a while ago. Should we change the original translations, @yoavf?